app/components/ui/home/TestimonialQuentin.tsx

```
@@ -0,0 +1,35 @@
+import Image from "next/image";
+import Balancer from "react-wrap-balancer";
+
+export default function TestimonialQuentin() {
+  return (
+    <section id="testimonial-quentin" aria-label="Testimonial by Quentin Lohri" className="mx-auto my-16 sm:my-24 w-full max-w-6xl px-3 pt-8">
+      <div className="mx-auto">
+        <figure className="mx-auto">
+          <blockquote className="mx-auto max-w-2xl text-center text-xl font-semibold leading-8 text-zinc-900 sm:text-2xl sm:leading-9 dark:text-zinc-50">
+            <p>
+              <Balancer>"It's clear, it's direct, it's simple. That's what I like about TofuPilot. I like things to be simple and pragmatic."</Balancer>
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)